tests/functional/spec/census_exit.spec.js

```diff
@@ -14,7 +12,7 @@ describe("Post submission exit", () => {
     $(HubPage.submit()).click();
     browser.deleteAllCookies();
     $(CensusThankYouPage.exit()).click();
-    expect(browser.getUrl()).to.equal(BASE_URL);
+    expect(browser.getUrl()).to.equal("https://surveys.ons.gov.uk/sign-in/");
```

Comment:
As discussed in Slack, this looks odd, but is as expect now surveys.ons.gov.uk is default when session is removed